presto-cli/src/main/java/com/facebook/presto/cli/Console.java

```diff
@@ -25,6 +25,7 @@
 import com.google.common.base.Throwables;
 import com.google.common.collect.ImmutableSet;
 import com.google.common.io.Files;
+import com.google.common.net.HostAndPort;
```

Comment:
Unused import. The IDE will fix these: Auto Import -> Optimize imports on the fly